src/components/button/Button.stories.jsx

```diff
@@ -70,4 +70,5 @@ Destructive.args = {
   size: 'large',
   label: 'Button',
   name: 'Destructive'
-};
\ No newline at end of file
+}; 
+
```

Comment:
nit: I think it would be good to not include this file in the PR since you don't actually change it

Reply:
how do make it so it doesnt include it?

Reply:
Manually removing the space and adding the change to a new commit should work I think.